src/output-json-dns.c

```diff
@@ -24,31 +24,17 @@
  */
 
 #include "suricata-common.h"
-#include "detect.h"
-#include "pkt-var.h"
 #include "conf.h"
 
-#include "threads.h"
 #include "threadvars.h"
-#include "tm-threads.h"
-
-#include "util-print.h"
-#include "util-unittest.h"
 
 #include "util-debug.h"
 #include "util-mem.h"
 #include "app-layer-parser.h"
 #include "output.h"
-#include "app-layer.h"
-#include "util-privs.h"
-#include "util-buffer.h"
-#include "util-proto-name.h"
-#include "util-logopenfile.h"
-#include "util-time.h"
 
 #include "output-json.h"
 #include "output-json-dns.h"
-#include "rust.h"
```

Comment:
But this does use rust, right ?

Reply:
It must come through another header.

Reply:
But we should keep `rust.h` here then ?
(no big deal anyways)